tutorials/plot_stats_spatio_temporal_cluster_sensors.py

```diff
@@ -191,6 +184,10 @@
 # ----------
 #
 # - What is the smallest p-value you can obtain, given the finite number of
-#    permutations?
+#   permutations?
 # - use an F distribution to compute the threshold by traditional significance
-#    levels. Hint: take a look at ``scipy.stats.distributions.f``
+#   levels. Hint: take a look at ``scipy.stats.distributions.f``
```

Comment:
`:class`: or maybe `:obj:` with single-backticks should make it link to the docstring for `f`